sqrl-planner/src/main/java/com/datasqrl/planner/graphql/GraphqlModelGenerator.java

```diff
@@ -200,10 +202,12 @@ protected void visitQuery(
     List<QueryParameterHandler> parameters = new ArrayList<>();
     for (FunctionParameter functionParameter : tableFunction.getParameters()) {
       final var parameter = (SqrlFunctionParameter) functionParameter;
-      parameters.add(
-          parameter.isParentField()
-              ? new RootGraphqlModel.SourceParameter(parameter.getName())
-              : new RootGraphqlModel.ArgumentParameter(parameter.getName()));
+      QueryParameterHandler queryParam;
+      if (parameter.isParentField()) queryParam = new ParentParameter(parameter.getName());
+      else if (parameter.isMetadata())
+        queryParam = new MetadataParameter(parameter.getMetadata().get());
+      else queryParam = new RootGraphqlModel.ArgumentParameter(parameter.getName());
```

Comment:
I don't think it's a good practice to omit curly braces in 1-line `for`s and `if`s, makes the code less readable.